### PR TITLE
depth data summary

### PR DESCRIPTION
**Pull request by mhines-usgs**
@jread-usgs some q's for you in line if you have a moment

### CODE REVIEW

File: configs/nldas_config.yml [hunk 1]
```diff
@@ -3,8 +3,8 @@ sub_variables: ['dlwrfsfc','pressfc','apcpsfc','vgrd10m','ugrd10m','dswrfsfc','s
 nldas_url: "dods://hydro1.sci.gsfc.nasa.gov/dods/NLDAS_FORA0125_H.002"
 data_times: ["1979-01-01","2015-12-31"]
 data_variables: ['dlwrfsfc','pressfc','apcpsfc','vgrd10m','ugrd10m','dswrfsfc','spfh2m','tmp2m']
-wps_url: "http://cida-test.er.usgs.gov/gdp/process/WebProcessingService"
```
Comment (jordansread) [inline configs/nldas_config.yml:6]: not a big deal, but revert changes to this file?

File: scripts/lake_summary_locations.R [hunk 2]
```diff
@@ -11,6 +11,11 @@ lake_summary_locations <- function(){
   
 }
 
+lat_from_id <- function(ids){
```
Comment (jordansread) [inline scripts/lake_summary_locations.R]: this file too

File: scripts/lake_summary_nldas.R [hunk 3]
```diff
@@ -80,7 +80,10 @@ lake_driver_nldas <- function(file='data/NLDAS_data/NLDAS_driver_file_list.tsv')
     times[2] <- parse_driver_file_name(post.files, 'time.end')
     times <- unname(sapply(times, function(x) paste0(substr(x,1,4),'-',substr(x,5,6),'-',substr(x,7,8), ' UTC')))
     ids <- parse_driver_file_name(post.files, 'ids')
+    lats <- lat_from_id(ids) # sort these, so we can get smaller chunks to process
```
Comment (jordansread) [inline scripts/lake_summary_nldas.R]: and this file

**Comment by jordansread** [comment]
rest looks good, but would be nice to revert those couple of files. 
Also, luke responded that all bathybase lakes have full bathymetry, so we can use that for the `type` variable.

**Comment by mhines-usgs** [comment]
Ok cool. I'll do the revert and merge later. Thanks Jordan.

**Pull request merged**